temporal_interceptor.go

```diff
@@ -0,0 +1,28 @@
+package otel
+
+import (
+	"github.com/roadrunner-server/sdk/v4/utils"
+	"go.opentelemetry.io/otel/propagation"
+	"go.opentelemetry.io/otel/trace"
+	"go.temporal.io/sdk/contrib/opentelemetry"
+	"go.temporal.io/sdk/interceptor"
+)
+
+// type alias for the interceptorscd
```

Comment:
What is `interceptorscd` ?

Reply:
Looks like tried to change catalog in terminal and by mistake added it to docblock 😮‍💨